test/unit/intrinsic/src/IntrinsicTest.cpp

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2020 Sergei Bastrakov
+ *
```

Comment:
@BenjaminW3 @psychocoderHPC what is the preferred convention of calling such test source files: with or without `Test` at the end? I believe alpaka uses both ways.

Reply:
I think you can kee it like it is.